lesson10/main.js

```diff
@@ -37,16 +37,22 @@ function getMenus(){
 			{to: "bookmark.html", img:"img/1.png", alt:"画像1", text: "ブックマーク"}, 
 			{to: "message.html", img:"img/2.png", alt:"画像2", text: "メッセージ"} 
 		]; 
-		setTimeout(() => {
-			resolve(menus);
-		}, 3000);
+			setTimeout(() => {
+				resolve(menus);
+			}, 3000);
 	})
 }
 	
-async function showMenus() {
-	const menus = await getMenus();
-	removeCircle();
-	renderMenus(menus);
+async function tryToGetMenus() {
+	try{
+		return await getMenus();
+	}catch(error){
+		console.error(error);
+	}finally{
+		removeCircle();
+	}
 }
 
-showMenus();
+tryToGetMenus().then((menus)=>{
+	renderMenus(menus);
+});

```

Comment:
```suggestion
async function initialize() {
     const menus = await tryToGetMenus();
     menus && renderMenus(menus);
};
initialize();
```
IMO
Since we are using try-catch-finally, the need to write a try theory seems weak if we receive it with then.
It is recommended that the "function that executes everything" be cut out and executed.

Reply:
I agree with sen-san!

Reply:
Thank you Sen-san :)
your code is beautiful .
Thanks to you, I learned how to code.
I fixed it in my own way.
Is it okay?
c0c544bb19269294ca0027f62142c5500a4f8256